develop/security-best-practices.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Security best practices
+description: Image security best practices guide
+keywords: docker, images, containers, vulnerability, cve
+redirect_from:
+  - /develop/scan-images/
+---
+
+As a developer, you can take a few simple steps to improve the security of your
+container. This includes:
+
+1. Choosing the right base image from a trusted source and keeping it small
+2. Using multi-stage builds
+3. Rebuilding images
+4. Check your image for vulnerabilities
+
+Now, let’s take a detailed look at each of these best practices:
+
+### Choose the right base image
+
+The first step towards achieving a secure image is to choose the right base
+image. When choosing an image, ensure it is built from a trusted source and keep
+it small.
+
+Docker Hub has more than 8.3 million repositories. Some of these images are
+[Official Images](../docker-hub/official_images.md), which are published by
+Docker as a curated set of Docker open source and drop-in solution repositories.
+Docker also offers images that are published by
+[Verified Publishers](../docker-hub/publish/index.md). These high-quality images
+are published and maintained by the organizations partnering with Docker, with
+Docker verifying the authenticity of the content in their repositories. When you
+pick your base image, look out for the **Official Image** and **Verified
+Publisher** badges.
+
+![Docker Hub Official and Verified Publisher images](images/hub-official-images.png){:width="700px"}
+
+When building your own image from a Dockerfile, ensure you choose a minimal base
+image that matches your requirements. A smaller base image not just offers
```

Comment:
```suggestion
image that matches your requirements. A smaller base image offers
```

@dvdksn feels a bit awkward, maybe not quite a typo ;)

Reply:
yeaaah, I didn't really take a pass on a lot of things in this document since it was just content I shuffled around from a previous place. Thanks for taking the time to check these bits!